pkg/kn/root/root_test.go

```diff
@@ -26,6 +27,7 @@ import (
 )
 
 func TestNewRootCommand(t *testing.T) {
+	os.Args = []string{"kn"}
```

Comment:
I wonder if `os.Args` should be returned back original state in `defer`, but it seems fine in test run.